css assignment/styles.css

```diff
@@ -13,11 +13,11 @@ nav {
   position: sticky;
```

Comment:
make navbar content to center
now it is moved on top end

Reply:
Done